content/articles/getting-started-with-burpsuite/index.md

```diff
@@ -0,0 +1,139 @@
+Burpsuite is one of the most popular web application security tools. Burp Suite is an intercepting proxy tool that allows us to intercept, analyze and modify requests coming from our browsers before they are sent to the remote server. With burp suite we can intercept HTTP messages, modify the header and body of a message, and manually test for vulnerabilities.
+
+Burpsuite is one of those important tools found in the arsenal of web application penetration testers, web application security researchers, CTF players, as well as bug bounty hunters. 
+
+![Proxy](/engineering-education/getting-started-with-burpsuite/illustration.png)
+
+Why Burpsuite?
+
+Burpsuite is not only a proxy tool. It is a master tool that can be used to perform a series of tasks like:
+
+- Web spidering 
+- Automated and Manual testing of web applications.
+- Web application analysis
+- Finding vulnerabilities 
+
+Another plus is its inbuilt chromium-browser.
+
+Burpsuite is available as a free community edition and a professional edition which costs about $3999 a year.
+
+In this article, we'll be going through the basic usage of burp suite.
+
+**Disclaimer:** This article is for educational purposes only.
+
+### Prerequisites
+
+- Linux or Windows OS
+- Mozilla Firefox Browser
+
+### Burpsuite Configuration
+If you are on Kali Linux, burp suite comes pre-installed. For other Linux distributions like Ubuntu, you'll need to download the community edition from [portswigger's website](https://portswigger.net/burp/releases/professional-community-2021-10-3?requestededition=community). Same applies to Windows OS.
+
+
+![Burpsuite download](/engineering-education/getting-started-with-burpsuite/download.png)
+
+Next, we open up Burpsuit. If you are on kali Linux, it can be found in the applications panel.
+
+![launch Burpsuite](/engineering-education/getting-started-with-burpsuite/burpsuite.png)
+
+We are presented with a window that has different options. Select `Temporary project` and click on `next`.
+
+![Temporary project](/engineering-education/getting-started-with-burpsuite/temp-proj.png)
+
+We'll stick with the default setting, so we click on `Start burp`.
+
+![Start burp](/engineering-education/getting-started-with-burpsuite/start-burp.png)
+
+Burpsuite has been launched successfully.
+
+![Burpsuite download](/engineering-education/getting-started-with-burpsuite/burp.png)
+
+Now, we need to set up the burp suite proxy. The burp suite proxy allows us to intercept a web request while it is being processed. We're simply instructing our browser to use burp suite as a proxy, allowing us to intercept and alter requests while surfing our website.
+
+Firstly, we need to install a firefox extension called [Foxy Proxy](https://addons.mozilla.org/en-US/firefox/addon/foxyproxy-standard/). Click `Add to Firefox` to install the extension.
```

Comment:
It is unnecessary to use Firefox, the reader can use any browser. Edit this, please.